selfservice/hook/web_hook.go

```diff
@@ -53,24 +59,43 @@ type (
 		url          string
 		templatePath string
 		auth         AuthStrategy
+		interrupt    bool
 	}
 
 	webHookDependencies interface {
 		x.LoggingProvider
 	}
 
 	templateContext struct {
-		Flow           flow.Flow          `json:"flow"`
-		RequestHeaders http.Header        `json:"request_headers"`
-		RequestMethod  string             `json:"request_method"`
-		RequestUrl     string             `json:"request_url"`
-		Identity       *identity.Identity `json:"identity"`
+		Flow           flow.Flow             `json:"flow"`
+		RequestHeaders http.Header           `json:"request_headers"`
+		RequestMethod  string                `json:"request_method"`
+		RequestUrl     string                `json:"request_url"`
+		Identity       *identity.Identity    `json:"identity,omitempty"`
+		Credentials    *identity.Credentials `json:"credentials,omitempty"`
```

Comment:
I don't think we should include these. I believe they would be empty always anyways and it would not be a good idea to send e.g. hashes over the network. The risk of accidental logging by some component or other types of compromise is just too high! So I say we remove this:

```suggestion
```

Reply:
Removed in d8511ebd